lib/engine/round/operating.rb

```diff
@@ -433,7 +433,7 @@ def buy_company(company, price)
         @just_sold_company = company
 
         @current_entity.companies << company
-        @current_entity.spend(price, entity)
+        @current_entity.spend(price, entity) if price.positive?
```

Comment:
this one shouldn't be here, i'll fix it

Reply:
$0 is not a valid price in 1889, but in a game where it is valid, this check will be needed.